fossunited/fossunited/doctype/foss_event_cfp_submission/foss_event_cfp_submission.json

```diff
@@ -160,25 +160,23 @@
   {
    "fieldname": "talk_description",
    "fieldtype": "Text Editor",
-   "label": "Talk Description",
-   "reqd": 1
+   "label": "Talk Description"
   },
   {
    "fieldname": "status",
    "fieldtype": "Select",
    "in_list_view": 1,
    "in_standard_filter": 1,
    "label": "Status",
-   "options": "Review Pending\nApproved\nRejected"
+   "options": "Open\nApproved\nRejected"
```

Comment:
I guess you are on a previous commit. I changed this from open to review pending before.

Reply:
Resolved